src/Mod/Draft/draftguitools/gui_shapestrings.py

```diff
@@ -57,8 +57,8 @@ def GetResources(self):
         """Set icon, menu and tooltip."""
 
         return {"Pixmap": "Draft_ShapeString",
-                "MenuText": QT_TRANSLATE_NOOP("Draft_ShapeString", "Shape from text"),
-                "ToolTip": QT_TRANSLATE_NOOP("Draft_ShapeString", "Creates a shape from a text string by choosing a specific font and a placement.\nThe closed shapes can be used for extrusions and boolean operations.")}
+                "MenuText": QT_TRANSLATE_NOOP("Draft_ShapeString", "ShapeString"),
```

Comment:
```suggestion
                "MenuText": QT_TRANSLATE_NOOP("Draft_ShapeString", "Shape From Text"),
```
Normal cad user doesn't know what a string is. 
@Roy-043 thoughts?

Reply:
Yes those names would be better!